src/core/Table/Table.tsx

```diff
@@ -121,6 +121,11 @@ export type TableProps<
    * Handler for when a row is clicked. Must be memoized.
    */
   onRowClick?: (event: React.MouseEvent, row: Row<T>) => void;
+  /**
+   * Modify the selection mode of the table
+   * @default 'multi'
+   */
+  selectionMode?: 'multi' | 'single';
```

Comment:
hmm.. I guess we cannot change now, but we could have used just a single prop for enabling selection and setting the mode. just now `isSelectable` name is not really good :/
`selectable?: 'single' | 'multi'`